source/products.html

```diff
@@ -19,6 +19,7 @@
   </nav>
   {% endif %}
 </div>
+<h1 class="visually-hidden">Products</h1>

```

Comment:
Might be best to use `{{ page.name }}` here instead of the hard-coded text, so it'll reflect the category or artist name if they're browsing either of those